### ISSUE TITLE
Compare blinded data vectors from KV-450 with Hildebrandt et al. (2018)

### ISSUE THREAD

**Issue by chihway** [LSSTDESC/txpipe-reanalysis#14]
Here's what we're trying to compare to: https://arxiv.org/abs/1812.06076

**Comment by ellongley** [comment]
Here is the latest comparison: 
<img width="495" alt="kv450" src="https://user-images.githubusercontent.com/25040359/85027754-4153e400-b148-11ea-98db-a3c2024c0827.png">

**Comment by MariaElidaiana** [comment]
Yellow is KV?

**Comment by ellongley** [comment]
Yes, apologies for not including the legend, the yellow is the blinded published results.

**Comment by ellongley** [comment]
![image](https://user-images.githubusercontent.com/25040359/86390836-ca017280-bc66-11ea-89e6-2010356ff226.png)
OK several changes yields better agreement but we aren't there yet, these were namely: 

1) using the updated weights (weight_recal) 
2) matching the photo-z binning to Hildebrandt et al. i.e. having z binning lower bound<Z_b<=upper bound rather than lower bound<=Z_b<upper bound (Z_B values can come in 0.01 increments, so this made a difference in the number density per bin)
